.changeset/thirty-moons-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-components": minor
+---
+
+Extend API of presentation table with unified selection.
```

Comment:
This is an API addition - could we explain what exactly was added and add an example for how to use it?